README.md

```diff
@@ -55,6 +55,8 @@ programmatically as well:
 | `wasm-tools component new` | [wit-component] | Create a component from a core wasm binary |
 | `wasm-tools component wit` |  | Extract a `*.wit` interface from a component |
 | `wasm-tools component embed` |  | Embed a `component-type` custom section in a core wasm binary |
+| `wasm-tools metadata show` |  | Show name and producer metadata in a component or module |
```

Comment:
You can call out the wasm-metadata crate here between the `|  |`

Reply:
Thanks